tests/test_table.py

```diff
@@ -1588,6 +1588,7 @@ def test_computed_col_exceptions(self, reset_db: None, test_tbl: catalog.Table)
         msgs = t.select(msg=t.add1.errormsg).collect()['msg']
         assert sum('division by zero' in msg for msg in msgs if msg is not None) == 10
 
+    @pytest.mark.skip('Crashes pytest')
```

Comment:
Please create a rough-edges ticket to investigate why this is happening

Reply:
https://pixeltable.atlassian.net/browse/PXT-638